client/src/components/new-request/search.vue

```diff
@@ -259,19 +266,13 @@ export default class NewSearch extends Vue {
   }
   @Watch('request_action_cd')
   watchRequestActionCd (newVal) {
-    if (Object.keys(bcMapping).includes(newVal) && ['BC'].includes(this.location)) {
-      let { value } = newReqModule.entityTypesBCData.find(ent => ent.rank === 1)
-      newReqModule.mutateEntityType(value)
-    }
-    if (Object.keys(xproMapping).includes(newVal) && ['CA', 'IN'].includes(this.location)) {
-      let { value } = newReqModule.entityTypesXPROData.find(ent => ent.rank === 1)
-      newReqModule.mutateEntityType(value)
-    }
     // Set default location to BC for the requests where BC is the only location option
     if (['CNV', 'MVE'].includes(newVal)) {
       this.location = 'BC'
+      this.jurisdictionDisabled = true
       return
     }
+    this.jurisdictionDisabled = false
```

Comment:
We might not need this property if the comment above is true.. though i could be wrong!